Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
-//Write code for the 1st task below this
-
-
-//Write your code above this line
-
-
-
-//Write code for the 2nd task below this
-
-
+//Write code for the 1st task below this

```

Comment:
The function name 'greet' should be more descriptive and follow proper naming conventions (e.g., 'greetPerson')